index.js

```diff
@@ -13,11 +13,17 @@ module.exports = requireStyle
 function requireStyle (name) {
   const path = styleResolve.sync(name)
   var css = readFileSync(path, 'utf8')
+  
+  const isWindows = /^win/.test(process.platform)
```

Comment:
i would rather we use a module for this, like [`is-windows`](https://github.com/jonschlinkert/is-windows)